<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGitc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="gridFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="2">
        <widget class="QLineEdit" name="leRepo"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="leOpts">
         <property name="toolTip">
          <string>See the GIT-LOG options for more information.</string>
         </property>
         <property name="placeholderText">
          <string>Type the log options here and press Enter to filter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnRepoBrowse">
         <property name="text">
          <string>&amp;Browse...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Repository:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbSubmodule">
         <property name="text">
          <string>Submodule:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="cbSubmodule"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="GitView" name="gitViewA" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="acQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="acAbout"/>
    <addaction name="acAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="acPreferences"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuIgnoreWhitespace">
     <property name="title">
      <string>&amp;Ignore whitespace</string>
     </property>
     <addaction name="acIgnoreNone"/>
     <addaction name="acIgnoreEOL"/>
     <addaction name="acIgnoreAll"/>
    </widget>
    <addaction name="acVisualizeWhitespace"/>
    <addaction name="menuIgnoreWhitespace"/>
    <addaction name="separator"/>
    <addaction name="acCompare"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="acCopy"/>
    <addaction name="acSelectAll"/>
    <addaction name="separator"/>
    <addaction name="acFind"/>
   </widget>
   <widget class="QMenu" name="menu_Merge">
    <property name="title">
     <string>&amp;Merge</string>
    </property>
    <widget class="QMenu" name="menuCopy_To_Conflict_Log">
     <property name="title">
      <string>Copy To Conflict &amp;Log</string>
     </property>
     <addaction name="acCopyLog"/>
     <addaction name="acCopyLogA"/>
     <addaction name="acCopyLogB"/>
    </widget>
    <addaction name="menuCopy_To_Conflict_Log"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Merge"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acQuit">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close &amp;Window</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
  <action name="acAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About QGitc</string>
   </property>
  </action>
  <action name="acPreferences">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionIgnore_whitespace_changes">
   <property name="text">
    <string>Ignore whitespace changes</string>
   </property>
  </action>
  <action name="acVisualizeWhitespace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Visualize whitespace</string>
   </property>
  </action>
  <action name="acIgnoreEOL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>At &amp;end of line</string>
   </property>
  </action>
  <action name="acIgnoreAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;All</string>
   </property>
  </action>
  <action name="acIgnoreNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;None</string>
   </property>
  </action>
  <action name="acCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="acSelectAll">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="acFind">
   <property name="icon">
    <iconset theme="edit-find">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="acCompare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Compare Mode</string>
   </property>
  </action>
  <action name="acShowGraph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;graph</string>
   </property>
  </action>
  <action name="acAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="acCopyLog">
   <property name="text">
    <string>From Current &amp;View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="acCopyLogA">
   <property name="text">
    <string>From &amp;A</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="acCopyLogB">
   <property name="text">
    <string>From &amp;B</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GitView</class>
   <extends>QWidget</extends>
   <header>gitview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leRepo</tabstop>
  <tabstop>btnRepoBrowse</tabstop>
  <tabstop>leOpts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
